usecase/interactor/pokemon_interactor.go

```diff
@@ -33,3 +33,14 @@ func (po *pokemonInteractor) GET(pokemons []*model.Pokemon) ([]*model.Pokemon, e
 
 	return po.pokemonPresenter.ResponsePokemons(pokemons), nil
 }
+
+//
```

Comment:
Add a description to the exported functions, this function and the one above

Reply:
Done